js/ui/date_box/ui.time_view.js

```diff
@@ -303,11 +328,6 @@ var TimeView = Editor.inherit({
         }
     },
 
-    _toggleDisabledState: function(value) {
```

Comment:
Why do we need to remove this method?

Reply:
This method is not necessary. To synchronize the `disabled` option, it is sufficient to create a nested component using the `createComponent` method

Reply:
Ok.